### ISSUE TITLE
Paper version of the schedule

### ISSUE THREAD

**Issue by BalestraPatrick** [swissmobidevs/appbuilders16#2]
Some people asked for a paper version of the schedule. Due to last-minute changes, it would be great to have a display for those who don't want to use the app or the website.

**Comment by seanlabastille** [comment]
Rather than printing lots of schedules it may be easier to use one of the lobby screens to show the latest schedule.

**Comment by BalestraPatrick** [comment]
@flufff42 Yes, exactly. It depends on the venue but @bontoJR already had a nice idea on how to improve this side 😄

**Comment by fpillet** [comment]
Lobby screens. Or better, IPAD PROS used as glorified displays.

Bonus: add a ticker line on the display, showing tweets tagged #AppBuilders

**Comment by BalestraPatrick** [comment]
@fpillet Cool idea! A nice idea would also be to show tweets or something else during breaks and talk switching on the main projector.

**Comment by fpillet** [comment]
That's an even better idea! Live display off tweets in-between sessions, give more dynamism to the pause times!

**Comment by bontoJR** [comment]
I can't share right now, but we are going to have something really interactive. 🙊

**Comment by akosma** [comment]
just like @fpillet I vote for iPad Pros with Panic's Status Board on it: http://panic.com/statusboard/
